tsconfig.json

```diff
@@ -3,12 +3,16 @@
         "lib": ["es2015", "es2016.array.include", "es2017.object"],
         "noEmit": true,
         "target": "esnext",
+        "module": "commonjs",
+        "moduleResolution": "node",
         "strict": true,
         "allowJs": true,
         "checkJs": true
     },
     "types": ["node"],
     "include": [
+        "./dev-tools/global.d.ts",
+        "./dev-tools/node.d.ts",
         "./dev-tools/globals.ts",
         "./sim/dex-data.js",
         "./sim/dex.js",
```

Comment:
Are you intentionally not adding the rest of `./sim/` to here?

Reply:
I added `./sim/*.js` and `tsc` still works.

Reply:
They're included by the <code>import</code> statements in <code>dev-tools/global.d.ts</code> so <code>./sim</code> files can actually be removed from there.